dotcom-rendering/src/components/StickyBottomBanner.importable.tsx

```diff
@@ -264,10 +284,11 @@ export const StickyBottomBanner = ({
 	}, [pageId, keywordIds]);
 
 	useOnce(() => {
+		if (!countryCode) return;
```

Comment:
When might `countryCode` be false-y here? I think the callback passed to `useOnce` will only be called if all dependencies are defined, and the `useCountryCode` hook ensures that nulls are converted to `undefined` (maybe this early return is just to make TS understand this?)

Reply:
It’s true that it should never be falsy, but alas TypeScript is not aware that this narrowing has happened inside the callback.

I’ve not managed to find get TS to understand this, and based on a recent proposal, we might be moving away from `useOnce` entirely: #9176